Dagens Nyheter.js

```diff
@@ -0,0 +1,230 @@
+{
+	"translatorID": "192a29de-3d6e-4850-984f-943764126429",
+	"label": "Dagens Nyheter",
+	"creator": "Sebastian Berlin",
+	"target": "https://www.dn.se/(nyheter|sok)/",
```

Comment:
The Target is a regular expression, i.e. special characters should be escaped and for efficiency you should mark the string beginning, i.e. here:
`^https?://www\.dn\.se/(nyheter|sok)/' (this is when you do it in Scaffold. If you write straight to the translator, you need double backslashes -- www\\.dn\\.se)

Reply:
Also, this looks like it might be too restrictive? e.g. 
https://www.dn.se/sport/fotboll/sverige-ute-ur-fotbolls-vm-forlust-mot-england-i-kvarten/
https://www.dn.se/ekonomi/ostkriget-skrammer-skanska-bonder/